docker/Dockerfile

```diff
@@ -90,8 +90,8 @@ RUN export SQLITE_AUTOCONF_ROOT=sqlite-autoconf-3420000 && \
     manylinux-entrypoint /build_scripts/build-sqlite3.sh
 
 COPY build_scripts/build-openssl.sh /build_scripts/
-RUN export OPENSSL_ROOT=openssl-1.1.1u && \
-    export OPENSSL_HASH=e2f8d84b523eecd06c7be7626830370300fbcc15386bf5142d72758f6963ebc6 && \
+RUN export OPENSSL_ROOT=openssl-3.0.8 && \
+    export OPENSSL_HASH=6c13d2bf38fdf31eac3ce2a347073673f5d63263398f1f69d0df4a41253e4b3e && \
```

Comment:
There's 3.0.9 now which fixes a couple of not-so-critical CVEs that 3.0.8 was exposed to.

Reply:
bumped to 3.0.10